ui/components/Filters.js

```diff
@@ -1468,13 +1474,13 @@ const ImportModal = React.memo((props) => {
 });
 
 const PublishModal = React.memo((props) => {
-  const { publishFormSchema, handleClose, handlePublish, title } = props;
+  const { handleClose, handlePublish, title } = props;
 
   return (
     <Modal
       open={true}
-      schema={publishFormSchema.rjsfSchema}
-      uiSchema={publishFormSchema.uiSchema}
+      schema={publishCatalogItemSchema}
+      uiSchema={publishCatalogItemUiSchema}
```

Comment:
thankyou for this, do make sure you remove the API code for fetching schema, also make sure that this schema goes through the function that publishFormSchema.rjsfSchema goes, we don't get meshmodels already in schema we update them in UI

Reply:
For technology field